proto/sources.proto

```diff
@@ -226,6 +229,7 @@ message GitHub {
   bool includePullRequestComments = 14;
   bool includeIssueComments = 15;
   bool includeGistComments = 16;
+  bool skip_binaries = 17;
```

Comment:
Slightly off topic but what is the convention for protos? Snake case, camel case, both?

Reply:
Argh, i should've probably just used camel case, even though the style guide does recommend snake.
https://protobuf.dev/programming-guides/style/#message-field-names

Reply:
I think I can actually update all of them to be snake_case. I'm fairly certain that wouldn't be a breaking change. I'll double check though.

Reply:
I thought the idea with protos was that the name doesn't matter, just the number. So I agree changing them shouldn't be a breaking change. The generated Go code would probably be the same too (hopefully).

Reply:
Fair enough. I was curious because I was about to add a new field and realized there was an inconsistency.

Reply:
Oh, yeah. I would also like to know (and stick to) whatever the convention is! Just commenting on the renaming idea.

Thanks for pointing out the inconsistency!